src/go/k8s/tests/e2e/redpanda-schema-registry/00-assert.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: apps/v1
```

Comment:
nit: can you rename the test folder to `schema-registry`, it should be probably implied that it's redpanda's? :) if we decide to integrate with other schema registries, those should probably have specific names